config/clock.rb

```diff
@@ -0,0 +1,10 @@
+require "clockwork"
+require_relative "boot"
+require_relative "environment"
+
+module Clockwork
+  every(1.day, "scheduled_messages.send", at: "19:30", tz: "UTC") do
+    puts "Sending scheduled messages"
+    MessageSender.new.delay.run
```

Comment:
Hey! OMG I just realized why -we need to add a worker process to the Procfile. SO glad you checked this out. I will try to write an integration test for it